WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -455,7 +445,7 @@ function(htmlArr, idx, item, field, colIdx, width, attr) {
 	htmlArr[idx++] = " id='";
 	htmlArr[idx++] = this._getCellId(item, field, params);
 	htmlArr[idx++] = "'";
-	var className = this._getCellClass(item, field, params);
+	var className = className || this._getCellClass(item, field, params);
```

Comment:
Classes will now be read from ZmListView.prototype._getCellClass.
Removed the className param